scripts/plugin-tests.js

```diff
@@ -2441,10 +2442,18 @@ let syncTests = {
         },
       ],
     })
+    const outputs = Object.keys(metafile.outputs).map(outputPath => {
+      const absolutePath = path.resolve(outputPath)
+      const relativePath = path.relative(testDir, absolutePath)
+
+      return relativePath
+    })
     const bundle = require(output)
     const result = bundle()
 
     assert.deepStrictEqual(result, shimData)
+    assert(outputs.includes('out.js'))
+    assert(outputs.some(outputPath => /in-([\w\d]+)-(\d)+\.js/.test(outputPath)))
```

Comment:
This regex checks for the format used in the filenames of the shim files — e.g. `in-{HASH}-{INDEX}.js`, where `in` is the name of the input file (`in.js`), `{HASH}` is a shasum of the shim file's contents, and `{INDEX}` is a numeric value containing the index of the dynamic import.